CHANGELOG.md

```diff
@@ -54,10 +54,11 @@ No changes to highlight.
 No changes to highlight.
 
 ## Full Changelog:
+* Images in the chatbot component are now resized if they exceed a max width by [@abidlabs](https://github.com/abidlabs) in [PR 2748](https://github.com/gradio-app/gradio/pull/2748)
+* Add Brazilian Portuguese translation (pt-br.json) by [@pstwh](http://github.com/pstwh) in [PR 2753](https://github.com/gradio-app/gradio/pull/2753)
```

Comment:
Final nit: Can you add an image here showing an example gradio demo translated to Portuguese! Would be really nice visual. If you edit the file directly on github, you just can drag and drop an image.

Thank you!